### ISSUE TITLE
[Bug] Not possible to properly use player actions

### ISSUE THREAD

**Issue by Miniontoby** [jaylinski/kodi-addon-soundcloud#65]
Hi,

I am here to report multiple issues I discovered that all are related to player actions.

- [ ] Seeking isn't possible:
  - If I seek to a point, it takes a while and then it sometimes just works, but most of the time it just:
    - skips to the next song
- [ ] Pausing doesn't work:
  - If I pause whilst not in the first x amount of seconds/minutes, it skips to the next song after x amount of seconds
- [ ] Skipping to the previous (and maybe also to the next) song doesn't always works:
  - If I skip to a previous song, kodi just crashes sometimes:
    - Crashlog: https://termbin.com/065v3
- [ ] Skips when bad internet connection:
  - Whenever my internet connection is bad it starts to stop a few times and continue and then it just skips to the next song

Here if my system info: 
```
 Arch: armv7l
 Kernel: Linux 6.1.58-v7l+ #1692 SMP Thu Oct 19 17:03:49 BST 2023
 Release: Raspbian GNU/Linux 11 (bullseye)
```

Kodi version: (`kodi --version`)
```
19.4 (19.4.0) Git:20221007-166303c4f0 Media Center Kodi
Copyright (C) 2005-2021 Team Kodi - http://kodi.tv
```

**Comment by jaylinski** [comment]
What audio format are you using? There are three different ones in the addon settings.

**Comment by Miniontoby** [comment]
The one you suggested over at https://github.com/jaylinski/kodi-addon-soundcloud/issues/61#issuecomment-1791036319

So: MP3 (HLS)

**Comment by jaylinski** [comment]
I can confirm the seeking issue, but only when using "mp3 (HLS)" format. The other formats work fine.

This could probably be resolved by supporting the inputstream-adaptive addon: https://github.com/xbmc/xbmc/issues/21267#issuecomment-1101174554

**Comment by Miniontoby** [comment]
Yeah, would be nice if this would be fixed... since I don't seem to be able to use any other format (afaik)
